torch/_prims/nvfuser_executor.py

```diff
@@ -45,7 +45,7 @@ def to_nvfuser(arg):
 
 
 # MyPy bug: https://github.com/python/mypy/issues/5107
-@lru_cache()  # type: ignore[arg-type]
+@lru_cache(maxsize=1024)  # type: ignore[arg-type]
```

Comment:
Would it be bad to use `max_size=None`, unlimited cache size (equivalent to Python 3.8's `functools.cache`)?

Reply:
maxsize=None doesn't seem to get the perf back to me neither. I'm now leaning towards not using `lru_cache` and going with a bare `Dict` instead. I'll ping you once I got the PR up.